index.html

```diff
@@ -411,33 +411,33 @@ <h3>Aggregated Credential Workflow</h3>
-              jobapplications.com - Verifier App->>Generic Verifier - Verifier Service:genericVerifer.com/presentations/verify?\"applicationwith A and B\"     
-              Generic Verifier - Verifier Service->>jobapplications.com - Verifier App:Verification Result     
-              jobapplications.com - Verifier App->>jobapplications.com - Verifier App:process application     
-              jobapplications.com - Verifier App->>Kenzie's Wallet - Holder Service:Application Accepted     
-              Kenzie's Wallet - Holder Service->>Kenzie - Holder:Application Accepted 
+              sequenceDiagram
+                  autonumber
+              Kenzie (Holder)->>jobapplications.com (Verifier App):jobapplications.com/presenations/submit?"application"
+              jobapplications.com (Verifier App)->>jobapplications.com (Verifier App):process application
+              jobapplications.com (Verifier App)->>Kenzie (Holder):Request credentials A and B
+              Kenzie (Holder)->>Kenzie's Wallet (Holder Service):Check for A and B
+              Kenzie's Wallet (Holder Service)->>Kenzie (Holder):Display A, B not found
+              Kenzie (Holder)->>Kenzie's Wallet (Holder Service):Need B
+              Kenzie's Wallet (Holder Service)->>backgroundcheck.com (Issuer App):backgroundcheck.com/index.html
+              backgroundcheck.com (Issuer App)->>Kenzie's Wallet (Holder Service):index.html
+              Kenzie's Wallet (Holder Service)->>backgroundcheck.com (Issuer App):backgroundcheck.com/getBackgroundCheck
+              backgroundcheck.com (Issuer App)->>Kenzie's Wallet (Holder Service):getBackgoundCheck.html
+              Kenzie's Wallet (Holder Service)->>backgroundcheck.com (Issuer App):backgroundcheck.com/getBackgrounCheck/credentials/issue?"RequestedInfo"
+              backgroundcheck.com (Issuer App)->>backgroundcheck.com (Issuer App):evaluate requested info
+              backgroundcheck.com (Issuer App)->>Generic Issuer (Issuer Service):genericissuer.com/credentials/issue?"backgrounCheckCredential+info"
+              Generic Issuer (Issuer Service)->>backgroundcheck.com (Issuer App):issued VC
+              backgroundcheck.com (Issuer App)->>Kenzie's Wallet (Holder Service):kenzieswallet.com/credentials/recieve?"VC"
+              Kenzie's Wallet (Holder Service)->>Kenzie (Holder):Notify B Ready
+              Kenzie (Holder)->>jobapplications.com (Verifier App):jobapplications.com/presenations/submit?"application"
+              jobapplications.com (Verifier App)->>Kenzie (Holder):Request Credentials A and B
+              Kenzie (Holder)->>Kenzie's Wallet (Holder Service):Send A and B to jobapplications.com
+              Kenzie's Wallet (Holder Service)->>jobapplications.com (Verifier App):jobapplications.com/presentations/submit?"application with A and B"
+              jobapplications.com (Verifier App)->>Generic Verifier (Verifier Service):genericVerifer.com/presentations/verify?"applicationwith A and B"
```

Comment:
```suggestion
              jobapplications.com (Verifier App)->>Generic Verifier (Verifier Service):genericVerifer.com/presentations/verify?"application with A and B"
```

Reply:
This item will be gotten with the full <br> update to the diagrams